gyrinx/core/static/core/scss/_classic_card.scss

```
@@ -673,6 +673,80 @@
 
+    // =======================================================================
+    // Text plates (#1816) — the gang card and the lore/notes cards. Same plate,
+    // nameplate and cog as the fighter card, but none of the fixed fighter
+    // regions (condition tabs, statline, weapons), so the title band moves up
+    // and the detail block owns the rest of the card.
+    // =======================================================================
+    &.classic-card--text {
+        // No condition tabs above the title band, and no statline below it.
+        --cc-name-top: 3.5mm;
+        --cc-body-top: 17mm;
+        // These plates are mostly picture and prose, so the portrait earns more
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)